setup.cfg

```diff
@@ -8,7 +8,7 @@ max-line-length = 127
 max-complexity = 26
 
 [isort]
-line_length=127
+line_length=1
```

Comment:
I'd strongly recommend just using isort's [built-in profile](https://pycqa.github.io/isort/docs/configuration/profiles/) for `black`